### PR TITLE
Splash and tournament image

### PR DESCRIPTION
**Pull request by MechanicKim**
Android
- Changed spalsh image(animation - delay(3 sec) repeat)
- To improve performance, use android native library
  I added a line on ‘build.gradle’ to add the library.

Common
- In tournament page, I added a ‘coming soon’ image and text.(Stacy’s req)

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -144,6 +155,8 @@ dependencies {
     compile 'com.google.firebase:firebase-messaging:10.0.1'
     compile 'com.google.android.gms:play-services:10.0.1'
     compile 'com.android.support:multidex:1.0.0'
+
```
Comment (ifndefdeadmau5) [inline android/app/build.gradle:158]: Did you add a line space here intentionally? or not...

File: src/components/Main.js [hunk 2]
```diff
@@ -124,6 +127,18 @@ const styles = StyleSheet.create({
     shadowOpacity: 0.5,
     shadowRadius: 3,
   },
+  comingView: {
```
Comment (ifndefdeadmau5) [inline src/components/Main.js]: How about change name to 'comingSoon...'? I couldn't understand the meaning of it at first

**Comment by ifndefdeadmau5** [comment]
Awesome! LGTM with few comment. Thanks

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/122/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

File: android/app/build.gradle (hunk 1 above)
Comment (MechanicKim) [reply android/app/build.gradle:158]: No! this is not my intention :-)

File: src/components/Main.js (hunk 2 above)
Comment (MechanicKim) [reply src/components/Main.js]: Okay! I'll change this name. Thanks!

**Comment by MechanicKim** [comment]
@ifndefdeadmau5
I resolved conflicts on my branch again! ^^

**Pull request merged**